realm/realm-library/src/main/java/io/realm/internal/ColumnIndices.java

```diff
@@ -50,4 +56,28 @@ public long getColumnIndex(Class<? extends RealmModel> clazz, String fieldName)
             return -1;
         }
     }
+
+    public ColumnIndices copyDeeply() {
```

Comment:
I changed my mind and now ColumnInfo is Cloneable c0eb1b6db67f7254ff9d86348eafeb3e960294cf